### PR TITLE
Bump to 0.22

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by hunchback** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by hunchback** [comment]
@safchain - when is the release expected?

**Comment by nplanel** [comment]
@hunchback we plan to do a release, the week before the hackathon

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.22.0] - 2019-02-21
+### Added
+- SRIO-V support
+- SFlow Counters support
```
Comment (lebauce) [inline CHANGELOG.md]: Add VPP probe

**Comment by hunchback** [comment]
pre condition merge: https://github.com/skydive-project/skydive/pull/1696

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by hunchback** [comment]
Please merge #1713 before bumping the version

**Comment by safchain** [comment]
@hunchback that's the plan

**Pull request merged**